includes/wikia/api/DWDimensionApiControllerSQL.class.php

```diff
@@ -32,6 +32,28 @@ class DWDimensionApiControllerSQL {
       ORDER BY city.city_id      
       LIMIT $limit';
 
+	const DIMENSION_USERS = '
+		SELECT 
+			user_id,
+			user_name,
+			user_real_name,
+			user_email_authenticated,
+			user_editcount,
+			STR_TO_DATE(user_registration, \'%Y%m%d%H%i%s\') AS user_registration,
+			CASE WHEN b.ug_user  IS NULL THEN false ELSE true END AS is_bot,
+			CASE WHEN bg.ug_user IS NULL THEN false ELSE true END AS is_bot_global,
+			CASE WHEN IFNULL(up.up_value,0) = 0 THEN false ELSE true END AS user_marketingallowed,
+			CASE WHEN up_skin.up_value IS NULL OR up_skin.up_value = \'\' THEN \'oasis\'    ELSE up_skin.up_value END AS user_skin,
+			CASE WHEN up_rte.up_value = 0 THEN \'source\' ELSE \'richtext\' END AS user_editor
```

Comment:
I don't really like this logic embedded in SQL queries.

Reply:
Ok, I can move it to code mapping!